project_timesheet_hour/models/project_project.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+
+    _inherit = 'project.project'
+
+    planned_hours = fields.Float(
+        string="Planned hours", compute='_compute_hours', store=True)
```

Comment:
Don't use the same compute method for the 3 fields, but instead call directly the other methods for the inheritance (although renaming to `_compute_*`).

Reply:
I created a method for each field.